spring-cloud-starter-stream-common-gemfire/src/main/java/org/springframework/cloud/stream/app/gemfire/config/GemfireSslProperties.java

```diff
@@ -15,41 +15,74 @@
 
 package org.springframework.cloud.stream.app.gemfire.config;
 
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.NotBlank;
+
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.core.io.Resource;
+import org.springframework.util.StringUtils;
+import org.springframework.validation.annotation.Validated;
 
 /**
  * @author Christian Tzolov
  */
 @ConfigurationProperties("gemfire.security.ssl")
+@Validated
 public class GemfireSslProperties {
 
 	private static final String USER_HOME_DIRECTORY = System.getProperty("user.home");
+	/**
+	 * Name of the local trust store file copied in the local file system.
+	 */
+	public static String LOCAL_TRUSTSTORE_FILE_NAME = "trusted.keystore";
+	/**
+	 * Name of the local trust store file copied in the local file system.
+	 */
+	public static String LOCAL_KEYSTORE_FILE_NAME = "keystore.keystore";
+
+	/**
+	 * Local directory to store the truststore and keystore files copied form the remote truststoreUri and getKeystoreUri uris.
+	 */
+	@NotBlank
```

Comment:
Not sure if this works, but in other places we use validation annotations on getters.

Reply:
Works well. Resolving...